### PR TITLE
Disable tests causing buffer overflow

### PR DESCRIPTION
**Pull request by chfast**

### CODE REVIEW

File: test/unittests/parser_test.cpp [hunk 1]
```diff
@@ -595,27 +596,29 @@ TEST(parser, milestone1)
...
     EXPECT_EQ(code1.instructions, (std::vector{Instr::loop, Instr::end, Instr::end}));
     EXPECT_EQ(code1.immediates.size(), 0);
 
     const auto loop_i32_empty = "037f0b0b"_bytes;
-    EXPECT_THROW(parse_expr(loop_i32_empty.data()), parser_error);  // loop arity != 0.
+    EXPECT_THROW_MESSAGE(
+        parse_expr(loop_i32_empty.data()), parser_error, "loop can only have type arity 0");
+}
+
+TEST(parser, DISABLED_instr_loop_input_buffer_overflow)
```
Comment (axic) [inline test/unittests/parser_test.cpp]: Why is this causing a buffer overflow?

Comment (chfast) [reply test/unittests/parser_test.cpp]: Because the end opcode is missing, and it continues until finds random byte which is a valid opcode.

Comment (axic) [reply test/unittests/parser_test.cpp]: As a reminder, `03400b` is consumed as `loop <0x40> end` and should fail with "loop can only have type arity 0". Why isn't that happening?

Comment (axic) [reply test/unittests/parser_test.cpp]: Ah it's missing the real `end` opcode.

Comment (axic) [reply test/unittests/parser_test.cpp]: Can you leave that as a comment?

Comment (chfast) [reply test/unittests/parser_test.cpp]: What exactly?

Comment (axic) [reply test/unittests/parser_test.cpp]: Don't know, tried to describe but its long: "The `loop` instruction is well formed as it contains the closing `end`, but the code block doesn't contain the implicit closing `end`."

Perhaps just add "`03400b0b`" would be valid.

File: test/utils/asserts.hpp [hunk 2]
```diff
@@ -15,4 +15,5 @@
     catch (...)                                                                              \
     {                                                                                        \
         ADD_FAILURE() << "Unexpected exception type thrown.";                                \
-    }
+    }                                                                                        \
+    (void)0
```
Comment (gumb0) [inline test/utils/asserts.hpp]: what does this do?

Comment (axic) [reply test/utils/asserts.hpp]: Ensure you need to add a trailing semicolon.

File: test/unittests/parser_test.cpp [hunk 3]
```diff
@@ -635,6 +639,13 @@ TEST(parser, instr_block)
         "09000000"_bytes);
 }
 
+TEST(parser, DISABLED_instr_block_input_buffer_overflow)
+{
+    // The function end opcode 0b is missing causing reading out of input buffer.
```
Comment (gumb0) [inline test/unittests/parser_test.cpp]:
```suggestion
    // The block end opcode 0b is missing causing reading out of input buffer.
```

Comment (gumb0) [reply test/unittests/parser_test.cpp]: Ah no, I misunderstood it, sorry

**Pull request merged**